_rules/printable-characters-shortcut-ffbc54.md

```diff
@@ -0,0 +1,483 @@
+      <br>
+      <button onclick="closeModal();">Dismiss</button>
+    </div>
+    
+    <input type="button" onclick="openModal()" value="Open modal" />
+    
+    <label for="target">Add to list (press "+" to add):</label>
+    <input type="text" id="target" />
+    <div>
+      To do list
+    </div>
+    <ul id="list"></ul>
+  </body>
+</html>
+```
+
+### Inapplicable
+
+#### Inapplicable Example 1
+
+This document is not an [HTML document][].
+
+```html
+<svg height="100" width="100">
+  <circle cx="50" cy="50" r="40" stroke="black" stroke-width="3" fill="red" />
+  Sorry, your browser does not support inline SVG.  
+</svg>
+```
+
+#### Inapplicable Example 2
+
+This [HTML document][] has a [keyboard event][] [dispatched][] to an [event target][] with the attribute `key` being a [printable character][] and the method `getModifierState` returning `true`.
+
+```html
+<html>
+  <head>
+    <title>Inapplicable Example 2</title>
+    <script src="/test-assets/ffbc54/shortcut.js"></script>
+  </head>
+
+  <body onload="registerShortcut({shortcutKey: '+', ctrlKey: true}); activateShortcuts();">
+    <label for="target">Add to list (press "ctrl" and "+" to add):</label>
+    <input type="text" id="target" />
+    <br />
+    <div>
+      To do list
+    </div>
+    <ul id="list"></ul>
+  </body>
+</html>
+```
+
+#### Inapplicable Example 3
+
+This [HTML document][] has a [keyboard event][] [dispatched][] to an [event target][] with the attribute `key` being a [non-printable character][non-printable characters].
+
+```html
+<html>
+  <head>
+    <title>Inapplicable Example 3</title>
+    <script src="/test-assets/ffbc54/shortcut.js"></script>
+  </head>
+
+  <body onload="registerShortcut({ctrlKey: true}); activateShortcuts();">
+    <label for="target">Add to list (press "ctrl" and "+" to add):</label>
+    <input type="text" id="target" />
+    <br />
+    <div>
+      To do list
+    </div>
+    <ul id="list"></ul>
+  </body>
+</html>
+```
+
+#### Inapplicable Example 4
+
+This [HTML document][] has a [keyboard event][] that is not [dispatched][] to an [event target][].
+
+```html
+<html>
+  <head>
+    <title>Inapplicable Example 4</title>
+    <script>
+      function shortcut() {
+        window.addEventListener('keydown', function(event) {
+          const target = document.getElementById("target");
+
+          if (event.key === "+") {
+            document.getElementById("list").innerHTML += "<li>" + target.value + "</li>";
+            target.value = "";
+            event.preventDefault();
+          }
+        });
+      }
+    </script>
+  </head>
+
+  <body onload="shortcut()">
+    <label for="target">Add to list (press "+" to add):</label>
+    <input type="text" id="target" />
+    <br />
+    <div>
+      To do list
+    </div>
+    <ul id="list"></ul>
+  </body>
+</html>
+```
+
+#### Inapplicable Example 5
+
+This [HTML document][] has an element with the attribute `accesskey`. Accesskeys use [non-printable characters][].
```

Comment:
Is it a non-printable character or a `getModifierState`?

Reply:
From what I could find out, accesskeys across the different browser/OS combinations all end up being different combinations of modifier keys. So, we can remove this example also.